dwh/es_data_marts/models/target_summary.sql

```diff
@@ -18,7 +18,11 @@ with
             sex_label,
             developmental_stage_label,
             disease_label,
-            significant
```

Comment:
> [!NOTE]
> The following comment applies both to **target_summary** and **gene_summary**, as the logic is very similar

I suggest that the output tables are removed for clarity:
* `target_summary` → `perturbed_gene_summary`
* `gene_summary` → `affected_gene_summary`